var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -36,6 +36,7 @@ class Magma(CMakePackage, CudaPackage):
     depends_on('blas')
     depends_on('lapack')
     depends_on('cuda@8:', when='@2.5.1:')  # See PR #14471
+    depends_on('cuda@:10.2.89', when='@:2.5.3')  # incompatible with CUDA 11
```

Comment:
```suggestion
    depends_on('cuda@:10', when='@:2.5.3')  # incompatible with CUDA 11
```
This might be better in case new CUDA 10.X.Y releases come out.

Reply:
Yes, that works. Thanks.